source/index.html.md

```diff
@@ -559,9 +559,11 @@ payment method must be omitted.
 </dl>
 
 <p class="alert alert-info">
-  <b>Notice:</b> An approved authorization that includes
-  <code>card[pares]</code> is 3-D
-  Secured and is strongly authenticated (SCA in PSD2).
+  <b>Notice:</b> A valid PARes included in <code>card[pares]</card> can indicate
+  3 different levels: non-authenticated, attempted 3-D Secure, fully 3-D Secure.
```

Comment:
I see we are inconsistent w.r.t. spelling numbers or writing them as digits, e.g. "Last 4 digits of card number." and "Our API offers six different resources:" :slightly_smiling_face:

No need to fix it in this PR.